build.gradle

```diff
@@ -368,7 +368,7 @@ allprojects {
       // TODO: these are temporary disabled (ethereum and sub modules), it should be removed in a future PR.
       '-org.hyperledger.besu.ethereum,' +
       '-org.hyperledger.besu.ethereum.*,' +
-      '-org.hyperledger.besu.ethstats.*,' +
+      //'-org.hyperledger.besu.ethstats.*,' +
```

Comment:
should this be removed rather than commented out

Reply:
I commented it out so that I can track them with other PRs that I will be bringing in. But you are right, we should remove this comment.